I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 222, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 101, in script
    with pause(1)
  File "game/script.rpy", line 101, in <module>
    with pause(1)
NameError: name 'pause' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 222, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 101, in script
    with pause(1)
  File "/Users/lynn/Pictures/2.Jennifer/1. Game characters/2. The actual game/renpy-6.99.13-sdk/renpy/ast.py", line 1282, in execute
    trans = renpy.python.py_eval(self.expr)
  File "/Users/lynn/Pictures/2.Jennifer/1. Game characters/2. The actual game/renpy-6.99.13-sdk/renpy/python.py", line 1794, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Users/lynn/Pictures/2.Jennifer/1. Game characters/2. The actual game/renpy-6.99.13-sdk/renpy/python.py", line 1788, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 101, in <module>
    with pause(1)
NameError: name 'pause' is not defined

Darwin-15.6.0-x86_64-i386-64bit
Ren'Py 6.99.13.2919
Test 1.0
